package.json

```diff
@@ -361,7 +367,7 @@
             "null"
           ],
           "default": null,
-          "description": "Specifies the full path to the OmniSharp server."
+          "description": "Specifies pre-release OmniSharp to use. Can be one of \"latest\", a specific version number, or the absolute path to a local OmniSharp folder."
```

Comment:
FWIW, I suspect that "pre-release" is not the right text since this can be set to any OmniSharp -- even those that are released. :smile: